scripts/run_backend_tests.sh

```diff
@@ -99,6 +99,9 @@ for arg in "$@"; do
   if [ "$arg" == "--generate_coverage_report" ]; then
     $PYTHON_CMD $COVERAGE_HOME/coverage combine
     $PYTHON_CMD $COVERAGE_HOME/coverage report --omit="$TOOLS_DIR/*","$THIRD_PARTY_DIR/*","/usr/share/*" --show-missing
+
+    echo "Generating xml coverage report..."
+    $PYTHON_CMD $COVERAGE_HOME/coverage xml
```

Comment:
Just to check -- should this be run instead of line 101, instead of in addition to it? Or do they do different things?

Reply:
Report shows the coverage on terminal. The one with:
```
extensions/__init__.py                                        0      0   100%
extensions/answer_summarizers/__init__.py                     0      0   100%
extensions/answer_summarizers/models.py                      72     41    43%   66-67, 70, 73-75, 92-93, 118-144, 160, 168, 184-187, 206-209, 228-231, 253-257, 279-293, 332-342
extensions/dependencies/__init__.py                           0      0   100%
```

Reply:
Ah, let me rephrase. Are we doing the same computation twice from scratch here? Or does "xml" reuse the coverage info that's already been generated?

Reply:
Yes xml use the coverage generated, i.e it uses the .coverage file.

Reply:
What does this command actually add? The command at line 101 already shows the coverage on the terminal. For reference, see [this PR's build](https://circleci.com/gh/lilithxxx/oppia/249) and [another PR's build](https://circleci.com/gh/oppia/oppia/11542).

Reply:
Code-climate works on the xml file. We have to generate a xml coverage if we need to make code-climate work. Its not for showing coverage on terminal

Reply:
Got it, thanks!